forms-flow-web/src/components/ServiceFlow/details/TaskHeaderListView.js

```diff
@@ -0,0 +1,364 @@
+import React, { useEffect, useState } from "react";
+ import { Row, Col } from "react-bootstrap";
+import {
+  getISODateTime,
+  getFormattedDateAndTime,
+} from "../../../apiManager/services/formatterService";
+import { useDispatch, useSelector } from "react-redux";
+import DatePicker from "react-datepicker";
+import moment from "moment";
+import "react-datepicker/dist/react-datepicker.css";
+import "./../ServiceFlow.scss";
+import AddGroupModal from "./AddGroupModal";
+import {
+  claimBPMTask,
+  fetchServiceTaskList,
+  getBPMTaskDetail,
+  unClaimBPMTask,
+  updateAssigneeBPMTask,
+  updateBPMTask,
+} from "../../../apiManager/services/bpmTaskServices";
+import { setBPMTaskDetailUpdating } from "../../../actions/bpmTaskActions";
+import UserSelectionDebounce from "./UserSelectionDebounce";
+import SocketIOService from "../../../services/SocketIOService";
+import { useTranslation } from "react-i18next";
+
+const TaskHeaderListView = React.memo(({task,taskId,groupView = true}) => {
+  const username = useSelector(
+    (state) => state.user?.userDetail?.preferred_username || ""
+  );
+  const taskGroups = useSelector((state) => state.bpmTasks.taskGroups);
+  const selectedFilter = useSelector((state) => state.bpmTasks.selectedFilter);
+  const reqData = useSelector((state) => state.bpmTasks.listReqParams);
+  const vissibleAttributes = useSelector((state) => state.bpmTasks.vissibleAttributes);
+  const firstResult = useSelector((state) => state.bpmTasks.firstResult);
+  const [followUpDate, setFollowUpDate] = useState(null);
+  const [dueDate, setDueDate] = useState(null);
+  const [showModal, setModal] = useState(false);
+  const [isEditAssignee, setIsEditAssignee] = useState(false);
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+  useEffect(() => {
+    const followUp = task?.followUp ? new Date(task?.followUp) : null;
+    setFollowUpDate(followUp);
+  }, [task?.followUp]);
+
+  useEffect(() => {
+    const due = task?.due ? new Date(task?.due) : null;
```

Comment:
when saving the task due date it will in UTC format right ?

Reply:
if in utc need to convert the utc into local date

Reply:
These time zone issues will change after testing need to verify from testing team